Table.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Table</title>
+    <style>
+        body{
+            font-family: sans-serif;
+            background-color: aliceblue;
+        }
+        section {
+            position: relative;
+            border: 1px solid #cac6c6;
+            padding-top: 37px;
+            background: #82dede;
+            margin-left: 100px;
+            margin-right: 100px;
+            border-radius: 3px;
+        }
+
+        .container {
+            height: 350px;
+        }
+
+        table {
+            border-spacing: 0;
+            width: 100%;
+        }
+
+        td {
+            border-left: 1px solid #eee;
+        }
+
+        td,
+        th {
+            border-bottom: 1px solid #eee;
+            background: white;
+            color: #000;
+            padding: 10px 25px;
+        }
+
+        th {
+            height: 0;
+            line-height: 0;
+            padding-top: 0;
+            padding-bottom: 0;
+            color: transparent;
+            border: none;
+            white-space: nowrap;
+        }
+
+        th div {
+            position: absolute;
+            background: transparent;
+            color: #fff;
+            padding: 9px 25px;
+            top: 0;
+            margin-left: -25px;
+            line-height: normal;
+            border-left: 1px solid #AAD9FD;
```

Comment:
Always use lowercase for the color code.

Reply:
Done